File: app/src/main/java/com/jkb/junbin/sharing/feature/message/util/DateUtil.java
package com.jkb.junbin.sharing.feature.message.util;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;

public class DateUtil {
    public static String getDateToString(long time) {
        Date d = new Date(time);
        SimpleDateFormat sf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.CHINA);
        return sf.format(d);
    }

}

File: Feature/message/src/main/java/com/jkb/junbin/sharing/feature/message/util/DateUtil.kt
package com.jkb.junbin.sharing.feature.message.util

import java.text.SimpleDateFormat
import java.util.*

object DateUtil {
    fun getDateToString(time: Long): String {
        val d = Date(time)
        val sf = SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.CHINA)
        return sf.format(d)
    }
}